Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -208,6 +206,21 @@ void GameFile::SaveCache()
   // TODO
 }
 
+QString GameFile::GetFileName() const
+{
+  return QFileInfo(m_path).fileName();
```

Comment:
Is it enough to move the "fix missing include" commit before the "change code that uses that include" commit? In that case, it seems obvious to me that it refers to preexisting code, since there's no other code before it. I'm having trouble thinking of a message to write that adds any useful information other than what's already there.

Anyways, this seems overly nitpicky.